mat/errors.go

```diff
@@ -114,7 +114,9 @@ type Error struct{ string }
 func (err Error) Error() string { return err.string }
 
 var (
+	ErrNegativeDimension   = Error{"matrix: negative dimension"}
```

Comment:
Should we fix the prefixes here or in another PR?

Reply:
Filed https://github.com/gonum/gonum/issues/1084